lightning/src/util/events.rs

```diff
@@ -1128,22 +1128,16 @@ pub enum MessageSendEvent {
 	/// Used to indicate that a channel_announcement and channel_update should be broadcast to all
 	/// peers (except the peer with node_id either msg.contents.node_id_1 or msg.contents.node_id_2).
 	///
-	/// Note that after doing so, you very likely (unless you did so very recently) want to call
-	/// ChannelManager::broadcast_node_announcement to trigger a BroadcastNodeAnnouncement event.
-	/// This ensures that any nodes which see our channel_announcement also have a relevant
-	/// node_announcement, including relevant feature flags which may be important for routing
-	/// through or to us.
+	/// Note that after doing so, you very likely (unless you did so very recently) want to
+	/// broadcast a node_announcement. This ensures that any nodes which see our
```

Comment:
Seems useful to link to `PeerManager::broadcast_node_announcement`, similar to before